lib/govuk_app_config/govuk_error/govuk_data_sync.rb

```diff
@@ -0,0 +1,56 @@
+require "time"
+
+module GovukError
+  class GovukDataSync
+    class MalformedDataSyncPeriod < StandardError
+      attr_reader :invalid_value
+
+      def initialize(invalid_value)
+        @invalid_value = invalid_value
+      end
+
+      def message
+        "\"#{invalid_value}\" is not a valid value (should be of form '22:00-03:00')."
+      end
+    end
+
+    class MissingDataSyncPeriod < StandardError
+      def message
+        "You must specify a ENV['GOVUK_DATA_SYNC_PERIOD'] (should be of form '22:00-03:00')."
+      end
+    end
+
+    attr_reader :from, :to
+
+    def initialize(govuk_data_sync_period)
+      raise MissingDataSyncPeriod if govuk_data_sync_period.nil?
+
+      parts = govuk_data_sync_period.split("-")
+      raise MalformedDataSyncPeriod, govuk_data_sync_period unless parts.count == 2
+
+      @from, @to = parts.map { |time| Time.parse(time) }
+    rescue ArgumentError
+      raise MalformedDataSyncPeriod, govuk_data_sync_period
+    end
+
+    def in_progress?
+      from.present? && to.present? && in_time_range?(from, to)
+    end
+
+  private
+
+    # `from`/`to` times are expected in UTC, based on:
```

Comment:
Thanks for adding this :-) might be good to tweak it slightly to "`from`/`to` times are done in the servers local time which is expected to be UTC, based on:"